lib/ocpp/v201/profile.cpp

```diff
@@ -0,0 +1,684 @@
+#include "ocpp/v201/profile.hpp"
```

Comment:
It might make sense to rename this file pair to `composite_schedule.*pp` or `charging_schedule.*pp`.

Reply:
Agreed. Right now the idea is to hold off on refactoring until later cards, and just match the names and patterns of v1.6.